common/src/main/java/com/wynntils/features/chat/ChatMentionFeature.java

```diff
@@ -56,10 +60,26 @@ protected void onConfigUpdate(Config<?> config) {
     }
 
     private Pattern buildPattern() {
-        return Pattern.compile(
-                "(?<!\\[)\\b(" + McUtils.mc().getUser().getName()
-                        + (!aliases.get().isEmpty() ? "|" + aliases.get().replace(",", "|") : "") + ")\\b(?!:|])",
-                Pattern.CASE_INSENSITIVE);
+        if (aliases.get().isEmpty()) {
+            return Pattern.compile(
+                    "(?<!\\[)\\b" + McUtils.mc().getUser().getName() + "\\b(?!:|])", Pattern.CASE_INSENSITIVE);
+        } else {
+            // we can filter the provided aliases and remove the invalid parts, then tell the user
+            String[] splitAliases = aliases.get().split(",");
+            List<String> filteredAliases = Arrays.stream(splitAliases)
+                    .filter(alias -> !alias.isEmpty())
+                    .map(alias -> FILTERED_CHARACTERS.matcher(alias).replaceAll(""))
+                    .toList();
+
+            if (splitAliases.length != filteredAliases.size()) {
+                McUtils.sendErrorToClient(I18n.get("feature.wynntils.chatMention.aliasesParseFailed"));
```

Comment:
The user doesn't know what got filtered?

Reply:
en_us for that mentions `Names should be separated with an single comma. Special characters (except underscore, dash, and space) will be filtered out.`, should we be more specific? This is really just meant so that people can put in their nicknames or whatnot, I don't think anyone is going to input `@#($*(@#$` and expect it to work

Reply:
hm okay, i guess fine as is then